lib/datadog/tracing/contrib/sinatra/tracer.rb

```diff
@@ -17,69 +17,12 @@ module Sinatra
         # Datadog::Tracing::Contrib::Sinatra::Tracer is a Sinatra extension which traces
         # requests.
         module Tracer
-          def route(verb, action, *)
```

Comment:
Not required any more

Reply:
Because we don't ask them to `register Datadog::Tracing::Contrib::Sinatra::Tracer`?